### ISSUE TITLE
NVDA cannot go through the multi select drop down by keyboard

### ISSUE THREAD

**Issue by crazyLayne** [nvaccess/nvda#11460]
### Steps to reproduce:
1. Open Chrome
2. Browse to https://codepen.io/layne_wright/pen/NWNKzRr?editable=true%3Dhttps%3A%2F%2Ffabricweb.z5.web.core.windows.net%2Foufr%2F7.124.2%2F
3. Open NVDA to visit the multi select drop down.
4. Press Enter to expand the drop down.
5. Press Down key to go through the options of drop down
(the narrator works well in this case)
### Actual behavior:
It skips the list of options

### Expected behavior:
It should announce the options and the status

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2020.2

#### Windows version:
OS Name	Microsoft Windows 10 Enterprise
Version	10.0.18363 Build 18363

#### Name and version of other software in use when reproducing the issue:
Version 84.0.4147.105 (Official Build) (64-bit)

#### Other information about your system:
Runing Windows 10 on a laptop

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by Adriani90** [comment]
This occurs probably because Chrome does not fire a focus redirect on the first option of the drop down after you expand it. If you switch manually to focus mode with nvda+space bar and then press enter on the drop down, then you can navigate with arrow keys the list of options as expected.
cc: @ObjectInSpace is there a bug on Chromium for this? What is the rationale for not redirecting the system focus to the first option of the drop down after it has been expanded?

**Comment by ObjectInSpace** [comment]
This issue only occurs for me if the "automatic focus mode for focus changes" box is unchecked. When it is checked, focus mode is activated in Chrome when pressing enter and the list of options is read and focused appropriately. Is this not the intended behavior for that setting? Or are people experiencing Chrome not focusing the options even when that setting is checked?

**Comment by Adriani90** [comment]
@ObjectInSpace, actually people are expecting the other way round as well. Even when that setting is unchecked and you navigate with NVDA in browse mode, the focus should still be redirected when expanding the list by pressing enter. Then the list could be rnavigated imediately after switching manually to focus mode with nvda+space bar.
However, I am not sure if this is possible to fix.

**Comment by Adriani90** [comment]
@crazyLayne are you still able to reproduce this issue with NVDA 2023.1 on Chrome or Edge 112?

**Comment by Adriani90** [comment]
We didn't hear any updates from the initial author and it seems the test case is not rendered properly in any of the browsers. We need an updated test case.
I am closing this for now as abandoned due to missing updates after we asked for it. If you are still having this issue, please comment with an updated test case and we can reopen.
